### PR TITLE
Update semantics.md intro

### PR DESCRIPTION
**Pull request by rat10**
I hope this is more understandable. The current version for me is hard to parse. I hope my proposed change is not erring on the too wordy side.

### CODE REVIEW

File: docs/semantics.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+There have been several different semantics proposals for quoted triples, some with multiple semantics.  The proposed semantics differ in technical details but also differ in the entailments they support.  The major differences between semantics can be summarized in whether they are sensitive to the precise syntax of quoted triples or only to the semantics of the components of quoted triples.   The first kind of semantics have been called opaque and the second kind transparent.  There is also a semi-opaque (or semi-transparent) semantics, as in the community report, where the treatment of blank nodes remains unchanged from standard RDF (i.e. transparent), but for IRIs and literals their precise syntactic form is significant.
```
Comment (TallTed) [inline docs/semantics.md:3]: Suggested changes to this paragraph are also found elsewhere... Please check and see whether those changes might be sufficient for you. If not, I have a little punctuation and language to suggest in yours, here --
```suggestion
There have been several different semantics proposals for quoted triples, some with multiple semantics.  The proposed semantics differ in technical details but also differ in the entailments they support.  The major differences between semantics can be summarized in whether they are sensitive to the precise syntax of quoted triples or only to the semantics of the components of quoted triples.   The first kind of semantics have been called opaque and the second kind transparent.  There is also a semi-opaque (or semi-transparent) semantics, as in the community report, where the treatment of blank nodes remains unchanged from standard RDF (i.e., they remain transparent, and their syntactic form remains unimportant), but for IRIs and literals, their precise syntactic form is significant.
```

Comment (rat10) [reply docs/semantics.md]: Thanks, IMO your suggestions improve the text!

Comment (TallTed) [reply docs/semantics.md]: See #66 and #71. The content of these three PRs overlaps a lot. Particularly, the content of docs/rdf-terminology.md and docs/semantics.md should be synchronized.

Comment (rat10) [reply docs/semantics.md]: It's three different issues: 1) naming (semi-transparent or semi-opaque), 2) the long explanation in semantics.md and 3) the short explanation in rdf-terminology.md. Issue 1 is waiting for comments, issue 2 is being worked on right now and after that is settled I might revisit issue 3 w.r.t. exact wording. Overlap is unavoidable as the topic is discussed in 2 documents, but to different purposes. (concise terminology lookup versus approachable explanation). My initial [comment ](https://github.com/w3c/rdf-star-wg/pull/66#issuecomment-1594909494) two weeks ago was a bit unfocused, but after that I think everything is going well.
